spec/tasks/whitehall_news_importer_spec.rb

```diff
@@ -108,4 +122,64 @@
     importer = Tasks::WhitehallNewsImporter.new
     expect { importer.import(parsed_json) }.not_to(change { Document.count })
   end
+
+  context "when an imported document has more than one edition" do
+    let(:import_published_then_drafted_data) do
+      {
+        content_id: SecureRandom.uuid,
+        editions: [
+          {
+            created_at: Time.zone.now,
+            news_article_type: { key: "news_story" },
+            translations: [
+              {
+                locale: "en",
+                title: "Title",
+                summary: "Summary",
+                body: "Body",
+                base_path: "/government/news/title",
+              },
+            ],
+            lead_organisations: [SecureRandom.uuid, SecureRandom.uuid],
+            supporting_organisations: [SecureRandom.uuid, SecureRandom.uuid],
+            worldwide_organisations: [SecureRandom.uuid, SecureRandom.uuid],
+            topical_events: [SecureRandom.uuid, SecureRandom.uuid],
+            world_locations: [SecureRandom.uuid, SecureRandom.uuid],
+            state: "draft",
+            force_published: false,
+          },
+          {
+            created_at: Time.zone.now - 1.day,
+            news_article_type: { key: "news_story" },
+            translations: [
+              {
+                locale: "en",
+                title: "Title",
+                summary: "Summary",
+                body: "Body",
+                base_path: "/government/news/title",
+              },
+            ],
+            lead_organisations: [SecureRandom.uuid, SecureRandom.uuid],
+            supporting_organisations: [SecureRandom.uuid, SecureRandom.uuid],
+            worldwide_organisations: [SecureRandom.uuid, SecureRandom.uuid],
+            topical_events: [SecureRandom.uuid, SecureRandom.uuid],
+            world_locations: [SecureRandom.uuid, SecureRandom.uuid],
+            state: "published",
+            force_published: false,
+            },
+          ],
+      }
+    end
+
+
+    it "sets the correct states when Whitehall document has more than one edition" do
+      parsed_json = JSON.parse(import_published_then_drafted_data.to_json)
+      Tasks::WhitehallNewsImporter.new.import(parsed_json)
+
+      expect(Document.last.publication_state).to eq("sent_to_live")
```

Comment:
It doesn't seem right that this is sent_to_live. I'd have thought it was a sent_to_draft matching https://github.com/alphagov/content-publisher/pull/421/files#diff-fc6876b3da5d8f577dec523eb02e8beeR148